common/styles/app.css

```diff
@@ -1299,7 +1299,7 @@ footer .container1 {
      display: none !important;
  }
 
- .modal-body .recordset-container alerts {
+ #record .modal-body .recordset-container alerts {
```

Comment:
Should this also apply to `recordedit`?

Reply:
Yes, I completely forgot about that. I did this change at the start of doing the changes and didn't care much about it. Now that I'm looking at it, I'm going to add extra class to the one that we want to show (recordsetSelectFaceting) and do the `display:none` for alerts that don't have that class name.